microservices/download/utils.js

```diff
@@ -38,6 +38,10 @@ function query_os(req) {
   let raw = req; // The URL string containing any number of query params.
```

Comment:
```suggestion
  let raw = req; // The query string containing any number of query params.
```

This would clarify it a bit for me, personally.

Reply:
Not a bad suggestion, I'll probably try to clean up this code just a little bit in the near future